### PR TITLE
Issue 149

### PR DESCRIPTION
**Pull request by psykokwak4**
- Reorganized `sdk/python` directory.
- Removed the technology sub directory in Python import statement.
- Enabled `pygen_test.py` (#140)
- Added a `--bundle` option to generate namespace packages.
- Added sanity tests and pygen tests for namespace packages.
- Added code to patch Python module for augmentation at run time .
- Combined `_yang_ns` from different packages to a single object at run time.
- Updated CI.

### CODE REVIEW

File: generate.py [hunk 1]
```diff
@@ -36,50 +39,64 @@ def init_verbose_logger():
...
+    version = ''
+    release = ''
+    setup_file = os.path.join(output_directory, 'setup.py')
+    with open(setup_file, 'r') as f:
+        for line in f:
+            if 'version=' in line or 'version =' in line:
+                rv = line[line.find('=') + 1:line.rfind(",")]
+                release = "release=" + rv
+                version = "version=" + rv[:rv.rfind(".")] + "'"
+    return release, version
```
Comment (abhikeshav) [inline generate.py]: Sorry if I missed this, but where are we generating a version file (`__version__.py`) to store the version? Why are we reading setup.py to get the version/revision?

File: sdk/python/profile/.includes/libnetconf.h [hunk 2]
```diff
@@ -0,0 +1,56 @@
+/**
+ * \file libnetconf.h
```
Comment (abhikeshav) [inline sdk/python/profile/.includes/libnetconf.h]: Should this be under profile or core?

File: test/travis_ci.sh [hunk 3]
```diff
@@ -185,39 +188,37 @@ function generate_ydktest_package {
...
-    run_test sdk/python/tests/test_sanity_errors.py native
-    run_test sdk/python/tests/test_sanity_filters.py native
-    run_test sdk/python/tests/test_sanity_levels.py native
-    run_test sdk/python/tests/test_sanity_filter_read.py native
-    run_test sdk/python/tests/test_sanity_netconf.py native
-    run_test sdk/python/tests/test_sanity_rpc.py native
-    run_test sdk/python/tests/test_sanity_delete.py native
-    run_test sdk/python/tests/test_sanity_service_errors.py native
-    run_test sdk/python/tests/test_ydk_client.py
+    run_test gen-api/python/tests/test_sanity_types.py native
```
Comment (abhikeshav) [inline test/travis_ci.sh]: Are we still gathering coverage for these files by running these tests from gen-api? The reason why these had to be run from sdk/python is that the files in sdk are checked into github and so the coveralls tool can gather the coverage and show it on coveralls.io

File: sdk/python/profile/.includes/libnetconf.h (hunk 2 above)
Comment (psykokwak4) [inline sdk/python/profile/.includes/libnetconf.h]: Sorry, all the "physical" files should reside in `core`.

File: test/travis_ci.sh (hunk 3 above)
Comment (psykokwak4) [inline test/travis_ci.sh]: Sure, sorry, did not know this, will change in next commit. What about coverage for generated model test file?

File: generate.py (hunk 1 above)
Comment (psykokwak4) [inline generate.py]: I removed the original `__version__.py` and found it easier read version from profile file and to modify the setup.py directly.

File: test/travis_ci.sh (hunk 3 above)
Comment (abhikeshav) [inline test/travis_ci.sh]: No problem. Coverage for generated model tests can be enabled once we integrate the model test cases to CI in [ydk-py](https://github.com/CiscoDevNet/ydk-py) repository.

**Pull request merged**